beta-20170105.md

```diff
@@ -0,0 +1,94 @@
+---
+title: What's New in beta-20170105
+toc: false
+summary: Additions and changes in CockroachDB version beta-20170105.
+---
+
+## Jan 5, 2017
+
+### Binaries
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-beta-20170105.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-beta-20170105.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+</div>
+
+### General Changes
+
+- CockroachDB now uses a more recent version of GRPC. [#9697](https://github.com/cockroachdb/cockroach/pull/9697)
+
+### SQL Language Changes
+
+- `FILTER` clauses for aggregate functions are now supported. [#10146](https://github.com/cockroachdb/cockroach/pull/10146)
+- Columns can now be created with collated string types, although these columns cannot yet be indexed or part of the primary key. [#12294](https://github.com/cockroachdb/cockroach/pull/12294)
+- `INTERVAL` values can now be added to `DATE` values. [#12428](https://github.com/cockroachdb/cockroach/pull/12428)
+- `TIMESTAMP` values can now be compared to `DATE` values. [#12431](https://github.com/cockroachdb/cockroach/pull/12431)
+- `extract()` now supports `DATE` values. As a consequence, `extract()` no longer supports `STRING` values, which must now be explicitly cast to `TIMESTAMP` or `DATE`. [#12479](https://github.com/cockroachdb/cockroach/pull/12479)
+- `DATE` literals in views now work correctly. [#12450](https://github.com/cockroachdb/cockroach/pull/12450)
+- Parsing of `INTERVAL` values is now more consistent with PostgreSQL. [#12505](https://github.com/cockroachdb/cockroach/pull/12505) [#12559](https://github.com/cockroachdb/cockroach/pull/12559) [#12566](https://github.com/cockroachdb/cockroach/pull/12566)
+- A new aggregate function `sum_int()` is now available. It is equivalent to `sum()` but its result is type `INTEGER` instead of `DECIMAL`.
```

Comment:
As @nvanbenschoten suggests, while discussion is ongoing, I'll remove this from the release notes in a follow-up PR.